packages/@react-stately/grid/src/useGridState.ts

```diff
@@ -7,7 +7,8 @@ export interface GridState<T, C extends GridCollection<T>> {
   /** A set of keys for rows that are disabled. */
   disabledKeys: Set<Key>,
   /** A selection manager to read and update row selection state. */
-  selectionManager: SelectionManager
+  selectionManager: SelectionManager,
+  disableNavigation: boolean
```

Comment:
Needs doc comment

Reply:
Also should probably be called `isNavigationDisabled` to match naming conventions.

And maybe something more specific? Like, what kind of navigation? `isKeyboardNavigationDisabled`??